_posts/05-03-01-Date-and-Time.md

```diff
@@ -6,59 +6,65 @@ anchor: date_and_time
 
 ## Datum i vreme {#date_and_time_title}
 
-PHP ima klasu sa nazivom DateTime koja služi da vam pomogne kada čitate, pišete, poredite ili računate datume ili vreme.
-Postoji mnogo funkcija koje se odnose na datum i vreme u PHP-u pored DateTime, ali ona pruža dobar objektno-orijentisani
-interfejs za najčešće upotrebe. Može da rukuje vremenskim zonama, ali to je van opsega ovog kratkog uvoda.
+PHP poseduje DateTime klasu za potrebe čitanja, pisanja, poređenja i računanja datume i vremena.
+Postoji i dosta drugih funkcija za rad sa datumom/vremenom u PHP-u pored DateTime klase, ali ona
+pruža dobar objektno-orijentisani interfejs za najčešće slučajeve korišćenja. Ova klasa može da
+manipuliše i vremenskim zonama, ali to je van okvira ovog kratkog uvoda.
+
+Da biste započeli sa radom sa DateTime klasom, konvertujte string koji sadrži datum i vreme u objekat pomoću
+`createFromFormat()` factory metode ili pozovite `new \DateTime` kako biste kreirali trenutni datum/vreme.
+Koristite `format()` metodu za konvertovanje DateTime instance u string.
```

Comment:
To sam namerno izaostavio, nekako mi delovalo suvišno da kežem "za konvertovanje u string za potrebe ispisa" ili kako već. Dovoljno je samo da se kaže da format služi za konvertovanje u string, ne mora da znači da će se obavezno posle toga ispisivati taj string.